src/rebar3_ex_doc.app.src

```diff
@@ -1,6 +1,6 @@
 {application, rebar3_ex_doc,
  [{description, "rebar3 plugin for generating docs with ex_doc"},
-  {vsn, "0.2.21"},
+  {vsn, "0.2.22"},
```

Comment:
Maybe I've already asked this, but why not use `semver` or `git` here?

Reply:
Hmm, we could use git I suppose. I've always found value in being exact, but it is yet another thing to keep with I suppose.